examples/deploy-container.sh

```diff
@@ -8,6 +8,11 @@ set -euo pipefail
 # Run with option `--interactive` or `-i` to start a shell for interacting with
 # the node.
 
+if [[ $EUID != 0 ]]; then
+    # NixOS containers require root permissions
+    exec sudo "PATH=$PATH" "NIX_PATH=$NIX_PATH" "NIX_BITCOIN_EXAMPLES_DIR=$NIX_BITCOIN_EXAMPLES_DIR" "${BASH_SOURCE[0]}" "$@"
```

Comment:
```
./examples/deploy-container.sh: line 13: NIX_BITCOIN_EXAMPLES_DIR: unbound variable
```

Reply:
Fwiw, it works if I remove `"NIX_BITCOIN_EXAMPLES_DIR=$NIX_BITCOIN_EXAMPLES_DIR"`.

Reply:
Fixed.